client/service/src/metrics.rs

```diff
@@ -172,33 +171,13 @@ impl MetricsService {
 		let mut timer = Delay::new(Duration::from_secs(0));
 		let timer_interval = Duration::from_secs(5);
 
-		// Metric and telemetry update interval.
-		let net_status_interval = timer_interval;
-		let net_state_interval = Duration::from_secs(30);
```

Comment:
Ah seems like net_state ticked once in 30 seconds, however now it ticks once in 5 seconds. Shall I fix it?

Reply:
I've been leaning towards removing sending this net state to the telemetry altogether.
If it's easy to fix, then please fix it. If it's difficult, I'd suggest to drop this piece of logic altogether and open an issue in the telemetry repo to remove the button.